tests/coverage/test_processor.py

```
@@ -0,0 +1,436 @@
+                                    </class>
+                                </classes>
+                            </package>
+                        </packages>
+                     </coverage>"""
+        root = ET.ElementTree(ET.fromstring(xml_str))
+        return root
+
+    monkeypatch.setattr(ET, "parse", mock_parse)
+
+
+class TestCoverageProcessorFactory:
```

Ruff (0.8.2)

Redefinition of unused `TestCoverageProcessorFactory` from line 18

(F811)